pyrightconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-    "include": ["mentat"],
-    "ignore": ["testbed", "tests", "scripts"],
-    "typeCheckingMode": "strict",
+  "include": ["mentat", "tests"],
```

Comment:
Added this for development which has helped catch a lot of edge-cases and bugs when updating the tests. I think a balance would be to just add non-strict typing to tests